src/Data/Impact.elm

```diff
@@ -184,27 +196,6 @@ scopeToString scope =
             "Textile"
 
 
-getPefWeighting : Unit.Ratio -> Unit.Ratio
-getPefWeighting weighting =
```

Comment:
Note to reviewers: removed because we never used it nor do we have any plan to in the future.